### ISSUE TITLE
Classification labels for drone detection

### ISSUE THREAD

**Issue by DanielCohenn** [ntu-aris/MMAUD#45]
Hey, 
I wanted to know if you have any classification labels (for example, 0 for no drone and 1 for drone) available for each image in the dataset, or if there is a way to easily generate them ?

EDIT: I read the Calibration drive section but I could not quite understand a few things:
- Is there a python script that performs the projection to generate the classification labels ?
- If I am doing so myself, how can I validate myself ? I am quite new to this and have no real experience in doing this.
- Have you used a YOLOV5 generated labels as your ground truth labels in the AV - FDTI paper ?

**Comment by snakehaihai** [comment]
Hi Daniel
I think we do share the labeled RECT somewhere in the issue sections. You may look after it. 

For the script, we don't really provide that. You may use the rect and combine with the CVPR anti-UAV challenge scripts. They mostly do 2D detection and in our case, most of our baseline are 3D trajectoy estimation so the requirment are different.  And I dont think it is hard to use copilot to generate such script for yolo training. 

Regards
Shenghai Yuan

**Comment by DanielCohenn** [comment]
Thank you for the response, 
I just want to know how did you generated the categorical labels for the classification head (C_{i}) in the AV - FDTI paper ? 
I would appreciate if could kindly walk me through this process. 

Thank you very much. 
@snakehaihai 

Edit: I can move this issue to the AV - FDTI repo if needed

**Comment by snakehaihai** [comment]
Hi Daniel,

Thanks for the question — the classification labels (C_i) come directly from the rosbag-level metadata.

During data collection, each individual rosbag contains only one drone flying (one drone class/ID per bag). When preparing the training set, we segment each rosbag into multiple shorter clips/windows, but we keep the original rosbag meta association. Therefore, every segment extracted from the same rosbag inherits the same categorical label C_i.

If you’d like, feel free to move this to the AV-FDTI repo — happy to continue the discussion there.
